ambari-server/src/main/java/org/apache/ambari/server/controller/internal/ArtifactResourceProvider.java

```diff
@@ -75,6 +76,8 @@
   public static final String CLUSTER_NAME_PROPERTY = RESPONSE_KEY + PropertyHelper.EXTERNAL_PATH_SEP + CLUSTER_NAME;
   public static final String SERVICE_NAME_PROPERTY = RESPONSE_KEY + PropertyHelper.EXTERNAL_PATH_SEP + SERVICE_NAME;
 
+  public static final String PROVISION_REQUEST_ARTIFACT_NAME = "provision_cluster_request";
```

Comment:
Minor concern: 

I think this change is fine the way it is, but we might want to consider refactoring this in the future.  

It may make more sense to keep any details about specific types of artifacts out of this class, and have a more generic strategy defined to allow for modifications of retrieved artifacts before the artifacts are returned to the caller.  

Even having a static list of registered "filtering" or "updating" interfaces in this class might be better, and allow the resource provider code to only interact with that strategy interface.

As I mentioned, this current change is fine, but we might want to consider refactoring this if we find that other artifact types need to be persisted, and then retrieved in some custom fashion.